webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -94,10 +94,13 @@ def timestamp_to_date(
 def time_tag(dt, format='%a, %b %d %H:%M %Z', future=False):
     if not isinstance(dt, (datetime.date, datetime.datetime)):
         try:
+            if isinstance(dt, unicode):
+                # isodate struggles to convert unicode strings with
+                # its parse_datetime() if the input string is unicode.
+                dt = dt.encode('utf-8')
```

Comment:
Mmm... I think there aren't any cases where there are valid unicode characters in an iso-standard date format. I think we should do `.encode('ascii')`. That will likely throw an encoding error for bad data, but only for bad data that is likely not a timestamp.